bot.js

```diff
@@ -102,10 +102,11 @@ bot.on('message', msg => {
   // Rank gold command
   else if (msg.content.startsWith(`${config.trigger}rank gold`)) {

```

Comment:
As mentionned, the word "gold" is only for internal uses. I'd rather name the command `$rank` instead of `$rank gold`